### ISSUE TITLE
Don't include multiple years in 'view data' from map

### ISSUE THREAD

**Issue by pviechnicki** [DataUSA/datausa-site#572]
It's confusing to see multiple years in the 'view data' pane on the maps.  It used to be we would only see one year's worth of data.  I'd recommend we go back to that as the default setting, and maybe add a control which says 'show all years'.

http://sunbird.datausa.io/map/?level=county&key=drug_overdose_deaths&translate=-288.1642054789155,4820.72643469117&scale=40812.177205666114

### Steps to Reproduce (for bugs)

1.
2.
3.

### Your Environment

* Browser Name:
* Operating System and Version:

**Comment by davelandry** [comment]
I don't agree with removing years from the table, and we never had that behavior in the site. It looks like the non-map data tables sort the values by year, while the map data table doesn't (which may have lead to this confusion). I'm implementing the sorting, which I believe helps this situation.

Keeping this issue open for further discussion.
